main.tf

```diff
@@ -338,3 +361,54 @@ module "wandb" {
     module.gke_app
   ]
 }
+
+# proxy-only subnet
+resource "google_compute_subnetwork" "proxy" {
+  name          = "${var.namespace}-proxy-subnet"
+  ip_cidr_range = var.ilb_proxynetwork_cidr
+  purpose       = "REGIONAL_MANAGED_PROXY"
+  role          = "ACTIVE"
+  network       = local.network.id
+}
+
+## In order to support private link required min version 0.13.0 of operator-wandb chart
+
+module "sleep" {
+  source = "matti/resource/shell"
+  version = "1.5.0"
+
+  environment = {
+    TIME = timestamp()
+  }
+  command              = "sleep 450; date +%s"
+  command_when_destroy = "sleep 450"
+  trigger              = timestamp()
+  working_dir          = "/tmp"
+
+  depends = [
+    module.wandb
+  ]
+}
+
```

Comment:
Why do we need this?

Reply:
To create a private link, we need to target an internal load balancer (forwarding rule). After deploying the secondary ingress via the Helm chart, it takes approximately 3 to 4 minutes for the internal load balancer to be created on GCP. Due to this delay, it's necessary to incorporate a sleep module with a duration of at least 450 seconds. This ensures that the private link resource does not fail during the provisioning process.